tests/test_store.py

```diff
@@ -307,7 +307,7 @@ def test_createTable(self, helpers, synapse_store, config, projectId, datasetId)
 
         # associate metadata with files
         manifest_path = "mock_manifests/table_manifest.csv"
-        inputModelLocaiton = helpers.get_data_path(get_from_config(config.DATA, ("model", "input", "location")))
+        inputModelLocaiton = helpers.get_data_path(config.model_location) 
```

Comment:
@andrewelamb I ran `TestTableOperations` locally and the tests weren't passing. When I printed out `inputModelLocaiton`, I saw: `/Users/lpeng/Documents/schematic-git2/schematic/tests/data/tests/data/example.model.jsonld` which is not correct. I think you could fix it by looking into `get_data_path` more and then correct the file path issue here. 

After you fix it, please ensure that you run the tests locally by doing `pytest -s tests/test_store.py::TestTableOperations`. I think these tests are not being run remotely as well. @GiaJordan could provide more context about these tests if needed.

Reply:
@linglp  By default `config.model_location` returns `tests/data/example.model.jsonld`. That is what is in the current config.yml file.  Is that not correct?

Reply:
Our unit tests used `test_config.yml` and the model path was relative, so it was specified as simply `example.model.jsonld`. For some reason the tests themselves are executed in the test directory but the schematic functions switch to the main schematic directory. If the tests are now using the regular config this is what's likely causing what @linglp described. The tests are needed, and there is an outstanding issue to set them up for running automatically on remote with the rest of our tests.

Reply:
doing this works:inputModelLocaiton = helpers.get_data_path(config.model_location)

Reply:
It should be, could you just confirm that the path is still correct when it is being used within schematic outside of the test functions?

Reply:
@GiaJordan I made that change, and the unit tests that @linglp asked to be run now pass, as do the ones in the testing workflow.

Reply:
perfect, thanks!